Source/RapyutaSimulationPlugins/Public/Sensors/ROS2CameraComponent.h

```diff
@@ -0,0 +1,87 @@
+// Copyright 2020-2021 Rapyuta Robotics Co., Ltd.
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "Camera/CameraComponent.h"
+#include "Components/SceneCaptureComponent2D.h"
+#include "Engine/TextureRenderTarget2D.h"
+
+#include "ROS2Node.h"
+#include <Msgs/ROS2ImageMsg.h>
+#include "ROS2Publisher.h"

```

Comment:
Just a note to avoid auto-reordering of header files by `clang-format`, by adding `//UE, //rclUE` on respective groups.